src/main/resources/wa-task-configuration-employment-et_scotland_multiple.dmn

```diff
@@ -166,7 +166,8 @@
           <text></text>
         </inputEntry>
         <inputEntry id="UnaryTests_1qondg8">
-          <text>"ReviewReferralJudiciaryMultiple"</text>
+          <text>"ReviewReferralJudiciaryMultiple",
+"ReviewReferralResponseJudiciaryMultiple"</text>
```

Comment:
Should there be an entry for Role Category, Description, the priorities, and dueDateIntervalDays as well?

Reply:
Added